mappings/net/minecraft/entity/EntityWithAiGoals.mapping

```diff
@@ -1,4 +1,17 @@
 CLASS us net/minecraft/entity/EntityWithAiGoals
-	FIELD a pos Ldx;
+	FIELD a aiCenter Ldx;
+	FIELD b aiRadius F
 	FIELD bv ID Ljava/util/UUID;
+	METHOD <init> (Lali;)V
+		ARG 0 world
+	METHOD a setAiRange (Ldx;I)V
+		ARG 0 center
+		ARG 1 radius
 	METHOD cO canSpawn ()Z
+	METHOD dd isInAiRange ()Z
+	METHOD de getAiCenter ()Ldx;
```

Comment:
I think `getAiHome` would be best. I want to keep `Ai` because this class is extended by lots of entities. Any time they reference the variable, I think its purpose would be more clear with `Ai` in it.